frontend/src/utils/api.ts

```diff
@@ -230,6 +230,9 @@ const api = {
       request: (options, template) => filteredpost(`/samples/prefill_template/`, {...options}, form({ template: template })),
     },
     search: q => get("/samples/search/", { q }),
+    sample_ids_by_default_selection_excepted_ids(defaultSelection: boolean, exceptedSampleIDs: Array<FMSSample['id']>, queryParams?: QueryParams) {
```

Comment:
why the different notation? if it's too long you can remove some typing ;).

Reply:
Just to add variety ;)